tools/fuzzer/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+project(SOF_FUZZER C)
```

Comment:
Having `project` here and `add_subdirectory(fuzzer)` in parent cmake lists is not good. You either should have it separated like testbench or just keep it as part of tools and remove `project` here.

Reply:
@jajanusz if I keep it separated, whats the best way to include it in the build when running scripts/build-tools.sh?

Reply:
Like this one https://github.com/thesofproject/sof/blob/f837e57b6e05c5386fd3fb43a4a6374d94475d47/scripts/host-build-all.sh#L8-L21